besu/src/test/java/org/hyperledger/besu/cli/BesuCommandTest.java

```diff
@@ -3684,7 +3698,7 @@ public void stratumMiningIsEnabledWhenSpecified() throws Exception {
   @Test
   public void stratumMiningOptionsRequiresServiceToBeEnabled() {
 
-    parseCommand("--miner-stratum-enabled");
+    parseCommand("--network", "dev", "--miner-stratum-enabled");
```

Comment:
Why was this needed?

Reply:
I needed to modify the test in a way that would ensure it was being run with a) `merge` enabled and b) `ethash` enabled, so that the warning message it was testing for would be produced. The default network is `mainnet.json` which doesn't have `ethash` enabled but `dev.json` does so I ended up going with that.